<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>aGrae GIS </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Busqueda de Explotaciones o Lotes</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>751</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0,0">
     <item row="0" column="4">
      <widget class="QDateEdit" name="sinceDate">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QDateEdit" name="untilDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>Desde:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="check_cultivos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Filtrar Cultivo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>Hasta:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="combo_cultivos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="line_buscar">
       <property name="inputMethodHints">
        <set>Qt::ImhUppercaseOnly</set>
       </property>
       <property name="placeholderText">
        <string>Nombre Lote o Explotacion...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="btn_reload">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>43</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4" rowstretch="0" columnstretch="0">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_add">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Id. Lote</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Id. Explotacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Id. Campania</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lote</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Explotacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Campa単a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Siembra</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cultivo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rinde Esperado [Kg/ha]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Produccion [Kg]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status Rinde</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Explotacion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Campa単a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="combo_exp"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_camp"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>771</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Monitor de Rendimiento</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Produccion Final:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_prod">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string>  Kg</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>43</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,0,0" columnstretch="0">
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_remove">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>681</width>
      <height>181</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Id. Lote</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Id. Explotacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Id. Campa単a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lote</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Explotacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cultivo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Campa単a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ajustar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_adjust">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Validar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_validate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>A単adir Lote al Mapa</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
